ConvertTo-Nuspec.ps1

```diff
@@ -0,0 +1,24 @@
+﻿param($Version)
```

Comment:
An extra build step costs effectively nothing. This way it's a formal extension point (that we can reuse if we ever need some of the more advanced nuspec features that `dotnet pack` doesn't provide) in the action instead of something custom-made and tacked-on. To my eyes, this makes it less hacky.

Reply:
Making this a public extension point feels like YAGNI to me, when the current use-case is very specialized, for a single purpose, nothing we would ever reuse.

On the other hand, maybe that's just how extensible software is built. And that's generally a good thing. Time will tell.